pkg/keystore/keystore.go

```diff
@@ -260,8 +247,8 @@ func NewTestKeystore() (*Keystore, string, error) {
 	if err != nil {
 		return nil, "", err
 	}
-	systemDir := filepath.Join(dir, common.DefaultSystemConfigDir)
-	localDir := filepath.Join(dir, common.DefaultLocalConfigDir)
+	systemDir := filepath.Join(dir, "/usr/lib/rkt")
```

Comment:
Ok, I consted them and added a comment. I am not sure about keeping them in sync because this is only for the unit tests. Does the comment I added work?